tensorrt_llm/serve/openai_protocol.py

```
@@ -1138,35 +1138,53 @@ def decode_opaque_state(encoded_opaque_state: Optional[str]) -> Optional[bytes]:
-        result.append([{
-            "token_id": tid,
-            "logprob": lp.logprob,
-            "rank": lp.rank
-        } for tid, lp in pos.items()])
+                f"first_gen_log_probs[{i}] must be a dict or float, got {type(pos)}"
+            )
     return result
 
 
 def _deserialize_first_gen_log_probs(
     serialized: Optional[List], ) -> Optional[list]:
```

Flake8 (7.3.0)

[error] continuation line with same indent as next logical line

(E125)